### ISSUE TITLE
YouTube Playlist AutoPlay - Issue

### ISSUE THREAD

**Issue by Giganotosaur** [crazyrabbit0/UserScripts#9]
Hey! So I found this script on reddit and every single comment on it was positive so i thought that it would help me out aswell but i didn't exactly so i tried it on different playlist and it worked but as i got closer to the end of the playlist it stopped wroking and It wasn't autoplaying. So i don't know what to do. Maybe I just messed up something but idk.  https://www.youtube.com/watch?v=UJ0Vu3LOYyA&list=PLaSOys9AMLlZ1fC-g3gg0wAORqxrwFDw2&index=402**** it sopped workin on this music idk what to do

**Comment by crazyrabbit0** [comment]
I don't get why it doesn't work for you, I tried your playlist and it goes all the way to 443 without a problem for me.
Is there a specific song where it stops, or is it random ?

**Comment by Arctoras** [comment]
It has stopped working for me as well when using my music playlist at somewhere around video 1080-ish. [https://www.youtube.com/playlist?list=PL14V4T43PcPkXJS_ZojMVLfoNqRa9SqLH]. I've had to go back to manually starting the next video each time.

**Comment by Arctoras** [comment]
I think the problem might be that I'm using ublock origin. It might be doing something unexpected.

**Comment by crazyrabbit0** [comment]
I'm also using uBlock Origin, maybe it's a specific filter list I don't use or another plugin.
Have you tried without plugins enabled ?

**Comment by Arctoras** [comment]
I've tried turning off all of my extensions other than tamper monkey, and it still didn't work. The only final thing I can think of is that I'm using Opera GX as my browser, so maybe that is doing something annoying. If Opera is the problem it is going to be annoying.

**Comment by crazyrabbit0** [comment]
Could be, not sure since I'm mainly using chrome

**Comment by TheDJTC** [comment]
Yeah I use firefox and playlists just kinda stop working around 400 which is why I was looking for a fix for that.

Tho on my end I had to hit refresh so that the script loaded (youtube is weird and simply going to another page doesn't count as "going to another page" aunno)

It works fine for me, so I dunno what to say besides checking to make sure it's showing the script when you're actually on youtube.

**Comment by Arctoras** [comment]
Yeah, it seems to not be running the script at all, which is fun. It might be some kind of problem with tamper monkey beta in that case. The annoying part is that it worked for a while and then suddenly stopped.

**Comment by crazyrabbit0** [comment]
Can't really help you there, I don't know why it wouldn't load the script.
Maybe try contacting the support team of the extension  🤔
